api/src/pcapi/routes/backoffice/templates/collective_offer_template/list.html

```diff
@@ -1,143 +1,135 @@
+{% extends "layouts/list.html" %}
+{% import "components/badges.html" as badges with context %}
+{% from "components/forms.html" import build_filters_form_ng with context %}
 {% import "components/links.html" as links with context %}
 {% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
 {% from "components/connect_as.html" import build_connect_as_link %}
-{% block page %}
-  <div class="pt-3 px-5 table-container-collective-offer-template-validation">
-    <h1>Offres collectives vitrine</h1>
-    {{ build_filters_form(form, dst) }}
-    <div>
-      {% if rows %}
-        <div class="d-flex justify-content-between">
-          <p class="lead num-results">
-            {{ rows | length }}
-            résultat{{ rows | length | pluralize }}
-          </p>
-          <div class="btn-group btn-group-sm"
-               data-toggle="pc-batch-confirm-btn-group"
-               data-toggle-id="table-container-collective-offer-template-validation-btn-group"
-               data-pc-table-multi-select-id="table-container-collective-offer-template-validation"
-               data-input-ids-name="object_ids">
-            <button disabled
-                    type="button"
-                    class="btn btn-outline-primary"
-                    data-use-confirmation-modal="true"
-                    data-modal-selector="#batch-validate-modal">Valider</button>
-            <button disabled
-                    type="button"
-                    class="btn btn-outline-primary"
-                    data-use-confirmation-modal="true"
-                    data-modal-selector="#batch-reject-modal">Rejeter</button>
+{{ build_connect_as_link(connect_as[venue.id], (public_name or venue.name) , "link-primary") }}
+{% set rows_count = rows|length %}
+{% block title %}Offres collectives vitrine{% endblock %}
+{% block search_block %}{{ build_filters_form_ng(form, dst) }}{% endblock %}
+{% block table_header %}
+  <th data-pc-column-name="Checkbox">
+    <input class="form-check-input fs-6"
+           type="checkbox"
+           name="pc-table-multi-select-check-all" />
+  </th>
+  <th>Actions</th>
+  <th>ID</th>
+  <th>Nom de l'offre</th>
+  <th>Formats</th>
+  {% if has_permission("PRO_FRAUD_ACTIONS") %}<th>Règles de conformité</th>{% endif %}
+  <th>État</th>
+  <th>Créateur de l'offre</th>
+  <th>
+    {% if date_created_sort_url %}
+      <a href="{{ date_created_sort_url }}"
+         class="text-decoration-none"
+         title="Changer pour un tri {{ 'croissant' if request.args.get("sort") == 'dateCreated' and request.args.get('order') == 'desc' else 'décroissant' }}">
+        Date de création
+        <i class="bi bi-sort-{{ 'down' if request.args.get("sort") == 'dateCreated' and request.args.get('order') == 'desc' else 'up' }}-alt"></i>
+      </a>
+    {% else %}
+      Date de création
+    {% endif %}
+  </th>
+  <th>Entité juridique</th>
+  <th>Partenaire culturel</th>
+  <th>Offres du partenaire culturel</th>
```

Comment:
Idée : La colonne est très large, on pourrait forcer :
```suggestion
  <th>Offres du<br/>partenaire<br/>culturel</th>
```

Reply:
j'ai surtout enelvé l'icone pour être cohérent avec les autres pages et quand on a un tableau plus chargé il se met bien en une petite colonne.